### PR TITLE
Cards adding implemented

### PR DESCRIPTION
**Pull request by matroskin062**

### CODE REVIEW

File: src/js/AddFormUI.js [hunk 1]
```diff
@@ -40,16 +42,19 @@ class AddFormUI {
...
+          description: '',
+          status: this.status,
+        };
 
-      this.input.value = '';
+        this.input.value = '';
 
-      CardAPI.createCard(newCard);
+        const card = await CardAPI.createCard(newCard);
+        new CardUI(card).renderCard(target.querySelector('.card-list'));
```
Comment (alexgul1) [inline src/js/AddFormUI.js]:
```suggestion
        CardAPI.createCard(newCard)
          .then(card => {
            new CardUI(card).renderCard(target.querySelector('.card-list'));
          });
```

i think this variant is better

File: src/js/AddFormUI.js [hunk 2]
```diff
@@ -40,16 +42,19 @@ class AddFormUI {
       this.addBtn.style.display = 'block';
     });
```
Comment (alexgul1) [inline src/js/AddFormUI.js]:
```suggestion
    this.addBtn.style.display = 'block';
    this.input.value = "";
    });
```

I think when you cancel the addition, the input should `""`

Comment (matroskin062) [reply src/js/AddFormUI.js]: yep, it makes sense

File: src/js/AddFormUI.js (hunk 1 above)
Comment (matroskin062) [reply src/js/AddFormUI.js]: Why you think so, any arguments?

Comment (alexgul1) [reply src/js/AddFormUI.js]: we will assign a value to the `card` variable only when the response returns and only then the next function will be called.
I think it is unnecessary to write the response to the variable if it is available in the `then` construction

Comment (matroskin062) [reply src/js/AddFormUI.js]: > we will assign a value to the `card` variable only when the response returns and only then the next function will be called.
> I think it is unnecessary to write the response to the variable if it is available in the `then` construction

It make sense, but not critical.
Finally both of cases works indenticaly.

Comment (matroskin062) [reply src/js/AddFormUI.js]: resloved in 9fb7e67

**Review by alexgul1 — APPROVED** [review]

**Pull request merged**